### DISCUSSION TITLE
How to generate static webpage and run it without a server

### DISCUSSION THREAD

**Discussion by bienieck** [dendronhq/dendron#3931] [Q&A]
I want to generate a static webpage and run it without a server. I want to zip it and send it to my colleagues and therefore I would like to avoid instructing them on how to set up a local server.

I know that [MkDocs](https://github.com/mkdocs/mkdocs/issues/526) as well as [Material](https://squidfunk.github.io/mkdocs-material/setup/building-for-offline-usage/) docs can generate such "offline" static files. How can I accomplish this with Dendron?

**Comment by levirs565** [comment]
## dendron-exports

Hello, i have create a project that can export Dendron vault notes into SSG-compatible Markdown named [dendron-exports](https://github.com/levirs565/dendron-exports)

My notes website is using dendron-export and lume. dendron-export will convert Dendron note into lume compatible markdown. After it, lume render markdown into HTML. You can see dendron-exports in action at https://notes.levirs.my.id/

Currently, dendron-exports only support [lume](https://lume.land/). I have created lume template that can use with dendron-export. Here is template link https://github.com/levirs565/dendron-lume .

You can request new SSG support by creating issue at dendron-exports or you can create your own renderer.

## How to generate static webpage and run it without a server?

You can use dendron-exports, lume and dendron-lume to generate static webpage. However, currently there are [lume bug](https://github.com/lumeland/lume/pull/451) that make generating static webpage not possible.
